app/build.gradle

```diff
@@ -67,7 +67,7 @@ dependencies {
     kapt(libraries.moshiCodegen)
     kapt(libraries.roomCompiler)
     implementation project(':store')
-    implementation project(':cache')
+    implementation 'com.nytimes.android:cache3:3.1.1'
```

Comment:
just a note, that repo is archived. We might want to consider maintaining source at some point